src/App/App.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import './App.scss';
+import { Link, Route, NavLink} from 'react-router-dom';
+
+class App extends Component<{}> {
```

Comment:
I definitely want to understand the necessity of `<{}>` better. Looks to be working the way we want though. Good stuff.